### DISCUSSION TITLE
请问现在最新版的电脑版洛雪音乐app怎么下载啊？看了一圈github也没明白，可以放一下百度连接吗？

### DISCUSSION THREAD

**Discussion by mggwxf12** [lyswhut/lx-music-desktop#1667] [Q&A]
请问现在最新版的电脑版洛雪音乐app怎么下载啊？看了一圈github也没明白，可以放一下百度连接吗？

**Comment by new123old** [comment]
https://github.com/lyswhut/lx-music-desktop/releases/tag/v2.5.0
在这里啊
